companion/lib/Variables/VariablesAndExpressionParser.ts

```
@@ -82,4 +89,119 @@ export class VariablesAndExpressionParser {
+
+		if (rawValue.isExpression) {
+			// Parse the expression
+			const parseResult = this.executeExpression(rawValue.value || '', undefined)
+			if (!parseResult.ok) throw new Error(parseResult.error)
+
+			return {
+				value: parseResult.value as any,
+				referencedVariableIds: parseResult.variableIds,
+			}
+
+			// nocommit - check value is valid according to the rules
```

ESLint

[error] Unexpected 'nocommit' comment: 'nocommit - check value is valid...'.

(no-warning-comments)